samples/outbox/sqltransport-nhpersistence-ef/SqlServer_2.2/Receiver/Program.cs

```diff
@@ -1,18 +1,22 @@
 ﻿using System;
+using System.Configuration;
 using NHibernate.Cfg;
 using NHibernate.Dialect;
 using NServiceBus;
 using NServiceBus.Transports.SQLServer;
 using NServiceBus.Persistence;
+using Configuration = NHibernate.Cfg.Configuration;
 
 class Program
 {
     static void Main()
     {
         Console.Title = "Samples.SQLNHibernateOutboxEF.Receiver";
-        using (var receiverDataContext = new ReceiverDataContext())
+
+        var connectionString = ConfigurationManager.ConnectionStrings["NServiceBus/Persistence"].ConnectionString;
```

Comment:
no point having connection configurable here. better just to hard code it in the cs